staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go

```diff
@@ -204,7 +203,7 @@ func NewApplyConflict(conflicts merge.Conflicts) *StatusError {
 			// TODO: Get obj details here?
 			Causes: causes,
 		},
-		Message: fmt.Sprintf("Apply failed with %d conflicts: %s", len(conflicts), conflicts.Error()),
+		Message: fmt.Sprintf("Apply failed with %d conflicts: %s", len(conflicts), message),
```

Comment:
While we're changing that, is there a good way to handle pluralization of "conflicts"?